src/packages/@ncigdc/components/Modals/ContinuousBinning/ContinuousCustomBinsModal.js

```diff
@@ -55,9 +57,9 @@ class ContinuousCustomBinsModal extends Component {
 
   componentDidMount = () => {
     const {
-      continuousBinType,
-      continuousCustomInterval,
-      continuousCustomRanges,
+      continuousBinType = 'default',
+      continuousCustomInterval = 0,
```

Comment:
Custom interval is an object now

{ Amount, max, min }

So it can be preserved in Redux when you switch to custom range bins